feature/experimental/system/gnmi/benchmarking/internal/setup/setup.go

```diff
@@ -329,6 +330,71 @@ func ConfigureATE(t *testing.T, ate *ondatra.ATEDevice) {
 	topo.StartProtocols(t)
 }
 
+// ConfigureOTG function is to configure ate ports with ipv4 , bgp
+// and isis peers.
+
```

Comment:
Remove the space here (and the comma after `ipv4`).

Reply:
probably this new commented line is better